### ISSUE TITLE
why not using kaggle instead of colab? There is P100, T4X2 for 36 hours in a week for free

### ISSUE THREAD

**Issue by search620** [TheLastBen/fast-stable-diffusion#258]
![image](https://user-images.githubusercontent.com/56544202/198263846-d6187fd1-66fd-4371-8cfc-f2124219bd00.png)

Am I missing something?

**Comment by joanorsky** [comment]
I must agree with this one. I am using kaggle for lots of works.. and it is a viable solution. the 2xT4 actually improves the generation by a lot comparing to colab ....

**Comment by TheLastBen** [comment]
Working on it, as soon as I'm done with new features in Dreambooth

**Comment by eran-barkai** [comment]
issue with kaggle is mounting the g-drive or equivalent...

**Comment by 1blackbar** [comment]
thats a big issue, but you can register multiple accounts and run at the same time so thats good, how do you guys download big files from it ?

**Comment by joanorsky** [comment]
you dont need to mount. You can share a model on your gdrive and then just use !Gdown --id <shared_link_id> and it downloads to your working area. The generated files can be zipped and download... easy peasy! Worth the trouble for the extra computational power...

The main differences is that all /content directories must be replaced by /kaggle/working .. and there are some details with gradio and streamlite .. (although they do work.. just not a direct port from colab to kaggle)

**Comment by aqualxx** [comment]
Has anyone got Kaggle working with fast-dreambooth? I'm getting an error when trying to run the training cell: `OSError: /opt/conda/lib/python3.7/site-packages/torch/lib/../../nvidia/cublas/lib/libcublas.so.11: undefined symbol: cublasLtHSHMatmulAlgoInit, version libcublasLt.so.11`

**Comment by txtyb** [comment]
> 

Same here.

**Comment by fritol** [comment]
> Has anyone got Kaggle working with fast-dreambooth? I'm getting an error when trying to run the training cell: `OSError: /opt/conda/lib/python3.7/site-packages/torch/lib/../../nvidia/cublas/lib/libcublas.so.11: undefined symbol: cublasLtHSHMatmulAlgoInit, version libcublasLt.so.11`

yes same here , tried a ton of tings and still this error

**Comment by TheLastBen** [comment]
I'm working on a kaggle version at the moment, will be available soon

**Comment by guumaster** [comment]
Any update on this? Looks like training V2 is not possible in free Colab, maybe in Kaggle is possible.

**Comment by mieucun123** [comment]
> I'm working on a kaggle version at the moment, will be available soon

Are there any updates?

**Comment by TheLastBen** [comment]
kaggle won't allow it,,even colab is limiting free use of SD

**Comment by mieucun123** [comment]
> kaggle won't allow it,,even colab is limiting free use of SD

It can run automatic111
